website/src/app/kb/authenticate/google/readme.mdx

```diff
@@ -1,56 +1,51 @@
 import Alert from "@/components/DocsAlert";
-import SupportOptions from "@/components/SupportOptions";
 import PlanBadge from "@/components/PlanBadge";
 import Image from "next/image";
+import Link from "next/link";
 
-<PlanBadge plans={["enterprise"]} />
+<PlanBadge plans={["enterprise"]}>
 
 # SSO with Google Workspace
 
-Firezone supports authenticating users to Google Workspace using our custom
-Google Workspace connector. Use this guide if you're looking to setup SSO with
-Google Workspace for your Firezone Enterprise account.
-
-<Alert
-  color="info"
-  html={`
-    Google Workspace integration is available in the Enterprise plan.
-    To authenticate users to your Google Workspace account in the Starter
-    plan <strong>without</strong> automatic user/group sync, use our
-    <a href="/kb/authenticate/oidc" class="text-accent-500 hover:underline">
-      universal OIDC connector
-    </a> instead.
-  `}
-/>
+</PlanBadge>
+
+Firezone integrates with [Google Workspace](https://workspace.google.com) using
+a custom connector that supports both authentication and directory sync. Use
+this guide if you're looking to setup SSO with Google Workspace for your
+Firezone Enterprise account and want to automatically sync users, groups, and
+organizational units from Google Workspace to Firezone.
+
+<Alert color="info">
+  If you're just looking to authenticate users against Google Workspace
+  **without** automatic directory sync, use our [universal OIDC
+  connector](/kb/authenticate/oidc) instead, available on all plans.
+</Alert>
 
 ## Overview
 
 The Firezone Google Workspace connector integrates with Google's identity APIs
-to support user authentication and user / group sync.
+to support user authentication and directory sync.
 
 Users, groups, and organizational units are synced every few minutes to ensure
 that your Firezone account remains up-to-date with the latest identity data from
-your Google Workspace account.
-[Read more about user / group sync here](/kb/authenticate/user-group-sync).
+Google Workspace. [Read more](/kb/authenticate/directory-sync) about how sync
+works.
 
 ## Setup
 
 Setting up the Google Workspace connector is similar to the process of setting
 up a universal OIDC connector. The main difference is the addition of a few
-extra scopes needed to enable user/group sync.
+extra read-only scopes needed to enable sync.
```

Comment:
I noticed you replaced `user / group` with `directory` above.  Would it make sense to put it here as well?